llvm/test/CodeGen/X86/inline-asm-memop.ll

```diff
@@ -0,0 +1,15 @@
+; RUN: llc -mtriple=x86_64-unknown-linux-gnu < %s
```

Comment:
Add FileCheck and run this through the update_llc_test_checks.py script to show the codegen